releasenotes/notes/defaults_TranspileConfig-4109a90c278d46df.yaml

```diff
@@ -0,0 +1,9 @@
+---
+upgrade:
+  - |
+    The ``TranspileConfig`` was restructured to include only the information
+    needed to construct a ``PassManager`` and it was renamed to ``PassManagerConfig``.
```

Comment:
> only the information needed to construct a ``PassManager`` 

can you elaborate on this

Reply:
I added the fields 34aec1e29. You think something else should be added?